spec/replace_browser_extension/02_user_interface_ios.rst

```diff
@@ -0,0 +1,197 @@
+---
+story: 60
+spec: 54
+shortcode: RBE
+title: Repalce Existing Browser Extension
+author: Dmitry Bespalov
+status: Final
+created: 2019-01-04
+---
+
+===============
+User interface
+===============
+
+.. contents:: Table of Contents
+
+1. Intro
+------------
+
+Intro screen, loaded with all information and ready to start.
+
+.. image:: screens/ios/RBE-Intro.png
+   :width: 320px
```

Comment:
The images are quite big. This make the Doc quite hard to digest. Can we make it more narrow?

Reply:
will experiment